player/ucb1_tuned/ucb1_tuned_node.py

```diff
@@ -10,5 +10,5 @@ def variance_bound(self):
 
     @property
     def eval(self) -> float:
-        ratio_of_visits = np.sqrt(np.log(self.parent.number_of_runs) / self.number_of_runs)
-        return self.sample_mean + np.sqrt(ratio_of_visits * min(0.25, self.variance_bound + 2 * ratio_of_visits))
+        ratio_of_visits = np.log(self.parent.number_of_runs) / self.number_of_runs
+        return self.sample_mean + np.sqrt(ratio_of_visits * min(0.25, self.variance_bound + np.sqrt(2 * ratio_of_visits)))
```

Comment:
I guess if we follow this formula:
![image](https://github.com/Michalski-Artur/MCTS-Connect-4/assets/79529984/87a19e99-ab34-488a-a5b6-f33edb4b5552)

We shouldn't take root here: `np.sqrt(2 * ratio_of_visits)`.
Also maybe we can use `c` parameter here?

Reply:
we have wrong formula in our doc

<img width="654" alt="image" src="https://github.com/Michalski-Artur/MCTS-Connect-4/assets/39590321/79d237b6-93a5-4156-a693-7c8aae2c3089">